### ISSUE TITLE
[Suggestion] Site-wide ban tweaks

### ISSUE THREAD

**Issue by reallytiredofclowns** [discuitnet/discuit#125]
It looks like a site-wide ban automatically logs out the user and they are then unable to log on.

When I've seen bans handed out like this on prod, the admins usually give a reason and ban duration (replying to the offending post or comment), if not permanent. If the user cannot log on, they can't access their notifications, and it's unlikely they'll read the post comments. I would therefore suggest restricting posting/commenting rights instead. Or, if DMs can be implemented quickly enough, to allow access to DMs while banned, so admins can explain the ban reasoning to the user.

**Comment by previnder** [comment]
Good idea. It was implemented that way because it was the simplest way to do it: just log out the user from all sessions and don't allow them log back in.

**Comment by Kaamkiya** [comment]
Also, redirecting them to `/@<username>` would work because it says "Account Suspended". So instead of being left wondering, or allowing them to log in, you could redirect them to the page that says that their account is banned.

Example of banned account page is https://discuit.net/@Spruce
